imap_processing/lo/l0/packet_definitions/ILO_APP_SHK.xml

```diff
@@ -0,0 +1,156 @@
+			</xtce:Parameter>
+			<xtce:Parameter name="TYPE" parameterTypeRef="uint1">
+				<xtce:LongDescription>CCSDS Packet Type Indicator</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SEC_HDR_FLG" parameterTypeRef="uint1">
+				<xtce:LongDescription>CCSDS Packet Secondary Header Flag</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="PKT_APID" parameterTypeRef="uint11">
+				<xtce:LongDescription>CCSDS Packet Application Process ID</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SEG_FLGS" parameterTypeRef="uint2">
+				<xtce:LongDescription>CCSDS Packet Grouping Flags</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SRC_SEQ_CTR" parameterTypeRef="uint14">
+				<xtce:LongDescription>CCSDS Packet Sequence Count</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="PKT_LEN" parameterTypeRef="uint16">
+				<xtce:LongDescription>CCSDS Packet Length</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SHCOARSE" parameterTypeRef="uint32">
+				<xtce:LongDescription>CCSDS Secondary Header MET</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="CDH_JUMPER_REG" parameterTypeRef="uint32">
+				<xtce:LongDescription>ILO_APP_SHK.CDH_JUMPER_REG</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="INSTR_ID" parameterTypeRef="uint8">
+				<xtce:LongDescription>ILO_APP_SHK.INSTR_ID</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="CDH_FPGA_VERSION" parameterTypeRef="uint8">
+				<xtce:LongDescription>ILO_APP_SHK.CDH_FPGA_VERSION</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="IFB_FPGA_VERSION" parameterTypeRef="uint8">
+				<xtce:LongDescription>ILO_APP_SHK.IFB_FPGA_VERSION</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="BULK_HVPS_VERSION" parameterTypeRef="uint16">
+				<xtce:LongDescription>ILO_APP_SHK.BULK_HVPS_VERSION</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="FSW_VERSION" parameterTypeRef="uint8">
+				<xtce:LongDescription>ILO_APP_SHK.FSW_VERSION</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="TABLE_VERSION" parameterTypeRef="uint8">
+				<xtce:LongDescription>ILO_APP_SHK.TABLE_VERSION</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SELECTED_CODE_IMAGE" parameterTypeRef="uint8">
+				<xtce:LongDescription>ILO_APP_SHK.SELECTED_CODE_IMAGE</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SELECTED_TABLE_IMAGE" parameterTypeRef="uint8">
+				<xtce:LongDescription>ILO_APP_SHK.SELECTED_TABLE_IMAGE</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="BOOT_COUNTER" parameterTypeRef="uint8">
+				<xtce:LongDescription>ILO_APP_SHK.BOOT_COUNTER</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="HV_LIMIT" parameterTypeRef="uint4">
+				<xtce:LongDescription>ILO_APP_SHK.HV_LIMIT</xtce:LongDescription>
```

Comment:
These descriptions aren't very... descriptive. Are these the ones in the spreadsheet?

Reply:
Yeah, that's all I got. That's actually from the short description, the long description is empty for most fields with descriptions like that. It looks like there are two that have better descriptions that I can update the xtce with, so I'll do that. And I didn't use the XML generator, I made these back in April and have been making small updates since then

Reply:
Use the `xtce_generator_template.py` that Tenzin just merged. We worked through this and it should use either the shortDescription or longDescription so you don't have to do it manually.